scripts/get-sources

```diff
@@ -161,9 +161,8 @@ fi
 if ! "$fresh_clone"; then
     echo "--> Merging..."
     git -c merge.verifySignatures=no merge --ff-only --commit -q "$VERIFY_REF"
-    if false && remote=$(exec git rev-parse --symbolic-full-name '@{u}' "$VERIFY_REF" 2>/dev/null) &&
-        [[ "$remote" =~ ^refs/remotes/ ]]; then
-        git update-ref -- "$remote" "$VERIFY_REF"
+    if git check-ref-format "refs/remotes/${GIT_REMOTE-}"; then
```

Comment:
Is this effectively a check for non-empty `$GIT_REMOTE` or am I missing something?

Reply:
This should check that the ref I will be updating exists.

Reply:
Ah, ok. So that call didn't but the current version does.